src/racing/actors.c

```diff
@@ -67,7 +69,7 @@ void cleanup_red_and_green_shells(struct ShellActor *shell) {
             	case GREEN_SHELL_HIT_A_RACER:
             	case BLUE_SHELL_LOCK_ON:
             	case BLUE_SHELL_TARGET_ELIMINATED:
-                	func_8000EE58(actorIndex);
+                	remove_actor_in_unexpired_actor_list(actorIndex);
```

Comment:
delete_
destroy_

_actor_in_active_actor_list

or

actor_list_delete_entry

Something like that.